### DISCUSSION TITLE
Raspberry 3b+ & RPI-RF-MOD & Externe SSD: Zu wenig Strom mit 2,5A-Netzteil?

### DISCUSSION THREAD

**Discussion by carusoe** [OpenCCU/OpenCCU#2199] [🇩🇪 Fragen&Antworten] [answered]
Hallo,

ich betreibe mein Homeassistant mit Raspberrymatic auf einen Raspberry 3B+ mit dem Funkmodul RPI-RF-MOD von ELV. In letzter Zeit schmiert mein System immer ab, da Schreib- und Lesefehler auf der SD-Karte auftreten. Daher meine Idee war es anstatt von der SD-Karte von einer externen SSD mit entsprechenden Case (SATA) zu booten.

Ich habe einige Recherchen abgeschlossen und habe mir eine Kingston SATA SSD mit einem Case UGREEN besorgt. Jedoch bekomme ich bei jedem Start einen "voltage error" und die SSD-Platte wird nicht erkannt ("no storage devices found"). Wenn ich aber von der SD-Karte starte, kann ich die Platte wunderbar mounten und Daten schreiben oder lesen (über HACLI).

Meine Frage ist nun: Zieht das RPI-RF-MOD so viel Strom, dass die externe SSD beim Starten nicht erkannt wird und wie kann man das Problem lösen?
 
Vielleicht hatte jemand das gleiche Problem und hat es schon gelöst. ;-)

**Comment by Baxxy13** [comment]
Eigentlich sollten die 2,5A reichen.
Wie versorgst du dein Setup?
Über das RPI-RF-MOD mittels Hohlstecker oder direkt den Pi3B+ mittels Micro-USB?

Sehr gut gehen auch die originalen Pi4B Netzteile (5.1V / 3A) die man mit einem Adapter (USB-C auf Hohlstecker oder USB-C auf Micro USB) nutzen kann und die definitiv genug "Dampf" haben.

**Comment by carusoe** [comment]
Hallo Baxxy13,

ich habe mit dem Modul das ELV-Gehäuse und das entsprechende Netzteil (Hohlstecker) mitbestellt, so dass der Raspi über das Modul mit Strom über den Hohlstecker versorgt wird.

Eben habe ich einfach mal das Funkmodul abgezogen und den Raspi 3B+ über das original 2,5A Netzteil angeschlossen. Siehe da, der gleiche Fehler.

@ALL: Kann mir jemand helfen, was ich tun kann? Der Raspi ist schon etwas älter (ich glaube ich habe ihn gekauft, kurz nachdem er rausgekommen ist). Vielleicht hilft ein Boot-PROM-Update oder so etwas? (Wie kann ich das herausbekommen?)

**Comment by carusoe** [comment]
Hallo nochmals,

ich habe jetzt alles ausprobiert. Immer kommt die gleiche Meldung unabhängig ob das RPI-RF-MOD-Modul installiert ist oder nicht. Zudem habe ich auch die Config.txt auf einer SD-Karte geändert und die Zeile program_usb_boot_mode=1.

Hier ist ein Screenshot:
![Boot_Screen](https://user-images.githubusercontent.com/97299861/215890940-f87e67fd-5f55-4ec0-8b81-22fcbb23704d.jpg)

Hat jemand noch eine Idee?

**Comment by carusoe** [comment, marked as answer]
Hallo,

ich habe es einfach mal mit einem aktiven Hub versucht. Und siehe da, es funktioniert. :-) Also war es doch ein Problem aus der Rubrik "Saftmangel". :-)
